### PR TITLE
Added a SQL question based on update query.

### PR DESCRIPTION
**Pull request by robinathon**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
 const sqlQuiz = [
+  {
+    Question: 
+      "How can the user change “codingWebsite” into “FreeCodecamp” in the “Name” column in the Users table?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:4]: Let's use single quotes here. We also need to use correct capitalization for freeCodeCamp. 

```suggestion
      "In SQL, how can the user change 'codingWebsite' into 'freeCodeCamp' in the 'Name' column for the Users table?",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    Answer: "UPDATE Users Set Name = 'FreeCodecamp' WHERE Name = 'codingWebsite' ",
+    Distractor1: "MODIFY Users Set Name = 'codingWebsite' WHERE Name = 'FreeCodecamp' ",
+    Distractor2: "UPDATE Users Set Name = 'codingWebsite' WHERE Name = 'FreeCodecamp' ",
+    Distractor3: "MODIFY Users Set Name = 'FreeCodecamp' WHERE Name = 'codingWebsite' ",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:8]: fixing incorrect spelling for freeCodeCamp.

```suggestion
    Answer: "UPDATE Users Set Name='freeCodeCamp' WHERE Name='codingWebsite' ",
    Distractor1: "MODIFY Users Set Name = 'codingWebsite' WHERE Name = 'freeCodeCamp' ",
    Distractor2: "UPDATE Users Set Name = 'codingWebsite' WHERE Name = 'freeCodeCamp' ",
    Distractor3: "MODIFY Users Set Name = 'freeCodeCamp' WHERE Name = 'codingWebsite' ",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    Explanation:
+      "The UPDATE command and WHERE clause are used together to change the name as asked in the question.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:10]:
```suggestion
      "The UPDATE command and WHERE clause are used together to change the name in a SQL table.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by robinathon** [comment]
Thank You for the suggestions @jdwilkin4 .
applied your suggested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**